### PR TITLE
Announcements — For #2972

### PR DESCRIPTION
**Pull request by WildcardSearch**
`announcements.php` Twig conversion for #2972

### CODE REVIEW

File: inc/views/base/announcements/announcements.twig [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.forum_announcement }}</title>
+{% endblock head %}
+
+{% block body %}
+
+<table border="0" cellspacing="{{theme.borderwidth}}" cellpadding="{{theme.tablespace}}" class="tborder tfixed clear no_bottom_border">
```
Comment (euantorano) [inline inc/views/base/announcements/announcements.twig]: Very small nitpick, but pesonally I like to have a space between the `{{` and the variable name 😄

**Comment by WildcardSearch** [comment]
Done @euantorano

**Comment by euantorano** [comment]
One more small thing @WildcardSearch. This line uses `htmlspecialchars_uni`: https://github.com/WildcardSearch/mybb/blob/9cf50b02a6eb56a5cf02fde0357e6114e3c7ac4c/announcements.php#L99

This will lead to the subject being escaped twice and displaying weird characters (Twig automatically escapes). The best approach is to remove the `htmlspecialchars_uni` on that line.

**Comment by WildcardSearch** [comment]
Done @euantorano 

So going forward, remove all `htmlspecialchars_uni()`, correct?

**Comment by euantorano** [comment]
@WildcardSearch Kind of. There are some cases where they may still be needed such as when passing user submitted content (usernames, posts, etc.) into a language variable that has HTML (some language variables have `<br />` and such in them). In that case, you need to still use `htmlspecialchars_uni` as we have to output the language variable with `|raw` (which disable's Twig's escaping).

**Pull request merged**